server/__init__.py

```diff
@@ -1,19 +1,18 @@
 from flask import Flask
+import os
 from flask_sqlalchemy import SQLAlchemy
 from flask_bcrypt import Bcrypt
 from flask_login import LoginManager, login_manager
```

Comment:
Just sorting imports:

```suggestion
import os

from flask import Flask
from flask_sqlalchemy import SQLAlchemy
from flask_bcrypt import Bcrypt
from flask_login import LoginManager, login_manager
```

First system imports, then external libraries, than internal imports.